### PR TITLE
SCAL-225472 added logger for sdk to handle debug flag

### PR DESCRIPTION
**Pull request by UtkarshRai12**

### CODE REVIEW

File: src/main/custom-chart-context.ts [hunk 1]
```diff
@@ -1112,6 +1114,6 @@ export const getChartContext = async (
     // wait for initialization here as the host app
     // needs to first handshake with the client app.
     await ctx.initialize();
-
+    // return null;
```
Comment (harshmeetTS) [inline src/main/custom-chart-context.ts:1117]: remove commented code if its not needed

File: src/main/util.ts [hunk 2]
```diff
@@ -9,3 +9,26 @@ export function timeout(promise: Promise<any>, ms: number, message?: string) {
...
+    if (paramKey === 'debug') {
+        if (paramValue === undefined || paramValue === null) {
+            return 'false';
+        }
+    }
+    if (paramValue === undefined || paramValue === null) {
+        return '';
+    }
+    return paramValue;
+}
```
Comment (harshmeetTS) [inline src/main/util.ts:26]:
```suggestion
export function handleMissingValue(
    paramKey: string,
    paramValue?: string | null,
): string {
    if (paramKey === 'debug') {
        return paramValue ?? 'false';
    }
  
    return paramValue ?? '';
}
```

**Review by harshmeetTS — APPROVED** [review]

File: src/main/custom-chart-context.ts (hunk 1 above)
Comment (UtkarshRai12) [reply src/main/custom-chart-context.ts:1117]: ack

File: src/main/util.ts (hunk 2 above)
Comment (UtkarshRai12) [reply src/main/util.ts:26]: ack

**Pull request merged**